src/api/spec/support/database_cleaner.rb

```diff
@@ -24,7 +24,7 @@
   config.before(:each) do |example|
     # For feature test we use truncation instead of transactions because the
     # test suite and the capybara driver do not use the same server thread.
-    if example.metadata[:type] == :feature
+    if example.metadata[:type] == :feature || example.metadata[:type] == :migration
```

Comment:
Why is this needed?

Reply:
Does not work with ``:transaction`` strategy. 

```
ActiveRecord::StatementInvalid: Mysql2::Error: SAVEPOINT active_record_1 does not exist: ROLLBACK TO SAVEPOINT active_record_1
```

Reply:
Created a dedicated commit for it to explain it!